R/filter_gene.R

```diff
@@ -1,14 +1,20 @@
 #' @title Filter Gene Expression File
 #' @description A simple function for additional gene expression filtering
-#' @param geneExp (Object) Name of expression data frame object generated using read_gene()
+#' @param expr (Object) Name of expression data frame object generated using read_gene()
 #' @param filter (Character) Filter option for additional filtering - either sum, var, or both
 #' @param threshold Numeric value between 0 and 1 for variance filter with default 0.1
+#' @param expr_file_out (Character) Filename of the base group to write out results to.  No file will
+#' be written when NULL.
 #' @return Returns filtered gene expression data frame to global env and writes to working directory
 #' @author Walid Nashashibi (\url{https://github.com/wnashash/})
 #' @examples
 #' \dontrun{
 #' library(processNetZoo)
-#' exp_path <- system.file("extdata", "expression_test.csv", package = "processNetZoo", mustWork = TRUE)
+#' exp_path <- system.file("extdata",
```

Comment:
R has a line length limit for documentation.  Broke up references to `exp_path` to stop the warning